.github/actions/spelling/patterns.txt

```diff
@@ -236,3 +236,6 @@ uuid_from_friendly_id!\(".*"\)
 
 # changelog.d fragment authors line
 ^authors: .*$
+
+# Pattern for PowerShell script
+(?i)\b\w+-Command\b
```

Comment:
```suggestion
(?i)\b-Command\b
```

I don't think you want leading "word characters", do you? I was also thinking this could be generalized to match any flag like:

```
# Pattern for matching CLI flags
`(?i)\b--?(\w)\b`
```